MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -5,7 +5,7 @@
 }
 
 <div class="login-content-wrapper">
-    <div class="login-phone-mode-homepage-wrapper">
+    <div class="login-phone-mode-homepage-wrapper" onclick="scrollLogin(900)">
```

Comment:
Sorry for that! 
At first I have tried using Jquery event handler, the reason why I still used **onclick** is to avoid extra coding. And importantly **$(element).click()** does not handle parameter while triggering function(since I need to pass _speed_).
I will figure it out and avoid using embedded script anymore.